### PR TITLE
Config the lifetime of temporary files

### PR DESCRIPTION
**Pull request by mh828**
Hello.
In order for the uploaded files to be adjustable in temporary mode, I made some changes in the program code that I don't think it was necessary to write a test for.

### CODE REVIEW

File: src/Features/SupportFileUploads/FileUploadConfiguration.php [hunk 1]
```diff
@@ -111,4 +111,9 @@ public static function maxUploadTime()
     {
         return config('livewire.temporary_file_upload.max_upload_time') ?: 5;
     }
+
+    public static function tempLifeTime()
```
Comment (PhiloNL) [inline src/Features/SupportFileUploads/FileUploadConfiguration.php:115]:
```suggestion
    public static function temporaryFileRetentionTime()
```

File: src/Features/SupportFileUploads/FileUploadConfiguration.php [hunk 2]
```diff
@@ -111,4 +111,9 @@ public static function maxUploadTime()
...
+    {
+        return config('livewire.temporary_file_upload.temp_life_time') ?: 1440;//in minutes
```
Comment (PhiloNL) [inline src/Features/SupportFileUploads/FileUploadConfiguration.php:117]:
```suggestion
        return config('livewire.temporary_file_upload.file_retention_time') ?: 60;
```

File: src/Features/SupportFileUploads/WithFileUploads.php [hunk 3]
```diff
@@ -106,7 +106,7 @@ protected function cleanupOldUploads()
             // of allFiles() to have already been deleted by another thread.
             if (! $storage->exists($filePathname)) continue;
 
-            $yesterdaysStamp = now()->subDay()->timestamp;
+            $yesterdaysStamp = now()->addMinutes(-1 * FileUploadConfiguration::tempLifeTime())->timestamp;
```
Comment (PhiloNL) [inline src/Features/SupportFileUploads/WithFileUploads.php:109]:
```suggestion
            $retentionTimestamp = now()->subMinutes(FileUploadConfiguration:: temporaryFileRetentionTime())->timestamp;
```

File: src/Features/SupportFileUploads/WithFileUploads.php [hunk 4]
```diff
@@ -106,7 +106,7 @@ protected function cleanupOldUploads()
...
             if ($yesterdaysStamp > $storage->lastModified($filePathname)) {
```
Comment (PhiloNL) [inline src/Features/SupportFileUploads/WithFileUploads.php:110]:
```suggestion
            if ($retentionTimestamp > $storage->lastModified($filePathname)) {
```

**Review by PhiloNL — CHANGES_REQUESTED** [review]
Please see the suggested changes. If you could add a test and documentation as well, that would be great.

**Comment by lrljoe** [comment]
@PhiloNL noticed a couple of instances in your updates that use addMinutes(-1*Q) rather than subMinutes(Q), just (politely!) wondering why that approach?

**Comment by PhiloNL** [comment]
> @PhiloNL noticed a couple of instances in your updates that use addMinutes(-1*Q) rather than subMinutes(Q), just (politely!) wondering why that approach?

That's exactly what I suggested in my review comments 😄 
https://github.com/livewire/livewire/pull/7286#discussion_r1420450699

**Comment by lrljoe** [comment]
> > @PhiloNL noticed a couple of instances in your updates that use addMinutes(-1*Q) rather than subMinutes(Q), just (politely!) wondering why that approach?
> 
> @lrljoe that's exactly what I suggested to be changed in my review comments 😄 [#7286 (comment)](https://github.com/livewire/livewire/pull/7286#discussion_r1420450699)

Think festivities have corrupted my eyes... definitely read it differently on mobile, cheers for doing the needful.  Hopefully @mh828 can make the relevant changes :)

**Comment by calebporzio** [comment]
Closing until there's more demand for this - it's a hard thing to test and anticipate issues so unfortunately it's not an easy merge for me. Thanks!

**Pull request closed without merging**